stdlib/3/multiprocessing/managers.pyi

```diff
@@ -20,7 +20,8 @@ _Namespace = Namespace
 class BaseManager(ContextManager[BaseManager]):
     address: Union[str, Tuple[str, int]]
     def connect(self) -> None: ...
-    def register(self, typeid: str, callable: Any = ...) -> None: ...
+    @classmethod
+    def register(cls, typeid: str, callable: Any = ...) -> None: ...
```

Comment:
I noticed that this function has a lot of other parameters in the implementation:
```
    @classmethod
    def register(cls, typeid, callable=None, proxytype=None, exposed=None,
                 method_to_typeid=None, create_method=True):
```
Is it worth adding those too? If they're not important, I can just merge this now and we'll worry about the additional parameters later.